ios/Podfile

```diff
@@ -0,0 +1,30 @@
+require_relative '../node_modules/react-native/scripts/react_native_pods'
+require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
+
+platform :ios, '11.0'
+
+target 'bootcamp' do
```

Comment:
pity that you didn't name it `Bootcamp` instead of `bootcamp` in the startup script ;)

Reply:
:(